modules/claims_api/spec/sidekiq/evidence_waiver_builder_job_spec.rb

```diff
@@ -38,15 +46,4 @@
       end
     end
   end
-
```

Comment:
Removing this test because one, the case is covered [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/requests/v2/veterans/evidence_waiver_request_spec.rb#L96), and two, the file number check happens in the controller.